scripts/entrypoint.sh

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
-
-if [[ -v USER_ID ]]; then
-	usermod -u $USER_ID coreboot
-fi
-if [[ -v GROUP ]]; then
-	groupmod -g $GROUP_ID coreboot
-fi
-
-runuser -u coreboot -- "$@"
```

Comment:
How does it work right now? I recall if somebody didn;t have their own user with ID 1000 on host, terrible things happened.

Reply:
It depends on what we call it and in what context. When I'm using it, the command looks as follows:

```sh
docker run --rm -it --user $(id -u):$(id -g) -v $PWD:/home/coreboot/coreboot ghcr.io/dasharo/dasharo-sdk:latest /bin/bas
```

[build.sh](https://github.com/Dasharo/coreboot/blob/dasharo/build.sh#L50) also seems to care about considering it.

What other use cases do we have?

Reply:
I don't know how it works TBH. I just recall problems with user ID mismatches.

@mkopec committed this file so maybe he has a better insight.

Reply:
@mkopec ping ?

Reply:
@pietrushnic I don't know the answer sadly, this piece of the file was copied from coreboot-sdk IIRC

Reply:
@mkopec Thanks, @miczyg1. What do we do in such a situation? What are the cases in which the script should work? I think that some leftovers are no longer valid in our use cases.

I'm using it in CI/CD and training VM. What other use cases should we confirm to clean up this PR and merge it finally?
We should have som simple regression here to ensure we don't break other use cases of this sdk.

Reply:
@pietrushnic haven't you had any troubles with permissions to directories after you cloned edk2? Is it perhaps related to the removal of this entrypoint?

Reply:
And wouldn't just calling `docker run` with `--user $(id -u):$(id -g)` eliminate the need of entrypoint?

Reply:
> @pietrushnic haven't you had any troubles with permissions to directories after you cloned edk2? Is it perhaps related to the removal of this entrypoint?

I had an issue, but it was my fault because I did a docker volume mount to an existing directory. In that case, Docker creates missing directories in the path with root privileges, leading to permission issues. I solved that by creating the correct directory before mounting the volume. But that is out of the scope of docker-sdk use. It was external to docker-sdk call.

> And wouldn't just calling docker run with --user $(id -u):$(id -g) eliminate the need of entrypoint?

Yes, that's why entrypoint is removed.